### ISSUE TITLE
Issue of columns when showing the error in HTML

### ISSUE THREAD

**Issue by mgueury** [mgueury/html_validator#5]
It seems that some page using or TABS or special characters like 
> septdir.ru 

Step-by-step
- go to septdir.ru 
- click on 1rst warnings
- the columns show issue at place where it is not.
It seems due to TABS in the HTML code.

**Comment by mgueury** [comment]
The problem happens also with the tidy binary and a very simplified testcase:

```
			<span data-uk-icon="icon: septdir-blog-web; ratio: 1.1">
                                             ^
```
mgueury@mgueury-VirtualBox ~/my_prog/html_validator/bug/issue_5 $ ./test.sh 
9	25	223	1	Warning: trimming empty <span>
ERROR: 0 WARNING: 1

Info: Document content looks like HTML5
<!DOCTYPE html>
<html>
<head>
<meta name="generator" content=
"HTML Tidy for HTML5 for Linux version 5.3.7">
<title>issue 5</title>
</head>
<body>
</body>
</html>

**Comment by mgueury** [comment]
Solution add tab-size 1 parameter when calling tidy.
Fixed in 0.982
